src/Ocis.php

```diff
@@ -58,8 +60,10 @@ class Ocis
         'This method is not implemented in this ocis version';
     private string $serviceUrl;
     private string $accessToken;
+    private string $educationAcessToken;
```

Comment:
this variable is only used when creating `$educationGuzzle`, so I believe it can be removed